straxen/plugins/peaklet_processing.py

```diff
@@ -574,15 +574,15 @@ def upper_rise_time_area_boundary(area, norm, const, tau):
         Function which determines the upper boundary for the rise-time
         for a given area.
         """
-        return norm*np.exp(area/tau) + const
+        return norm*np.exp(-area/tau) + const
 
     @staticmethod
     def upper_rise_time_aft_boundary(aft, slope, offset, aft_boundary, flat_threshold):
         """
         Function which computes the upper rise time boundary as a function
         of area fraction top.
         """
-        res = 10**(-slope * aft + offset)
+        res = 10**(slope * aft + offset)
```

Comment:
sorry @zhut19, still the slope is initialized as "-1", so we don't need to change, no?

Reply:
I think the current version is the correct one is not it? Or what do you mean @skazama ?

Reply:
Okay now I got it sorry.